jekyll/_cci2/ssh-access-jobs.md

```diff
@@ -18,19 +18,23 @@ This document describes how to access a build container using SSH on CircleCI 2.
 Often the best way to troubleshoot problems is to SSH into a build container and inspect 
 things like log files, running processes, and directory paths. CircleCI 2.0 gives you the option to access all jobs via SSH.
 
-When you log in with SSH, you are running an interactive login shell. You are also likely to be running the command on top of the directory where the command failed the first time, so you are not starting a clean run. In contrast, CircelCI uses a non-interactive shell for running commands by default. Hence, steps run in interactive mode may succeed, while failing in non-interactive mode. 
+When you log in with SSH, you are running an interactive login shell. You are also likely to be running the command on top of the directory where the command failed the first time, so you are not starting a clean run. In contrast, CircelCI uses a non-interactive shell for running commands by default. Hence, steps run in interactive mode may succeed, while failing in non-interactive mode.
```

Comment:
I don't think this was updated in this PR but CircleCI was spelled as `CircelCI` here.